### DISCUSSION TITLE
[Suggestion] Implement playerbot diff

### DISCUSSION THREAD

**Discussion by brian8544** [mod-playerbots/mod-playerbots#191] [Ideas]
This isn't the right place to make such request, but I feel like this would be a great optimization, which allows to run more bots.

Add a diff with/without player in server. The server will tune bot activity to reach the desired server tick speed (in ms), as what MaNGOS is using

https://github.com/celguar/mangosbot-bots/commit/ce4bd79d2dbc7e8453303f9f94c27761615c64a9

**Comment by liyunfan1223** [comment]
That's a good idea, prepare to look into it.
